card/mod/05_standard/set/type/search_type.rb

```diff
@@ -188,8 +192,8 @@ def default_search_params
 format :rss do
   view :feed_body do |args|
     case raw_feed_items args
-    when Exception ; @xml.item { render :search_error }
-    when Integer   ; @xml.item { render :search_count }
+    when Exception ; @xml.item( render :search_error )
```

Comment:
`{ } `does not work but `( )`. Not sure whether it is related to library upgrade or historical problems